forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -2669,8 +2671,22 @@ public static void addSpellAbility(final KeywordInterface inst, final CardState
             sbAttach.append(" | Bestow$ True | ValidTgts$ Creature");
 
             final SpellAbility sa = AbilityFactory.getAbility(sbAttach.toString(), card);
-            sa.setDescription("Bestow " + ManaCostParser.parse(cost) +
-                    " (" + inst.getReminderText() + ")");
+            final StringBuilder sbDesc = new StringBuilder();
+            sbDesc.append("Bestow");
+            final Cost bCost = new Cost(cost, false);
+            final boolean onlyMana = bCost.isOnlyManaCost();
+            String remTxt = inst.getReminderText();
+            if (!onlyMana) {
+                String oText = card.getCard().getOracleText();
+                String bestowStr = oText.substring(indexOf(oText, "Bestow"), indexOf(oText, "\\n"));
+                if (bestowStr.contains("(")) {
+                    String s = bestowStr.substring(indexOf(bestowStr, "(") + 1, indexOf(bestowStr, ")"));
+                    if (!s.equals("")) remTxt = s;
+                }
```

Comment:
The looking at the OracleText doesn't look nice to me

maybe we can change it later to something better?

Reply:
Yeah, I thought if we had some more templating examples we would have a better place to start.

Just having the one with the non mana cost is a bit weird